### DISCUSSION TITLE
Keep Watch > Star > Fork order

### DISCUSSION THREAD

**Discussion by crazy-max** [community/community#8599] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/1951866/145443502-84738d51-2969-46ed-8e06-3d008e4d52b8.png)

![image](https://user-images.githubusercontent.com/1951866/145441340-b671a15a-db21-412e-aa5a-ef1674d5f54d.png)

With this new feature Fork and Star buttons are swapped which I find quite disturbing.

**Comment by atom0s** [comment]
Is there a reason we keep seeing these changes just pushed live and not previewed? 

What is the point of the Preview setup if you guys don't use it for things that are actually annoying like this and the color change to closed issues?

**Comment by freysie** [comment]
I find this quite disturbing too. At the risk of sounding like an old person yelling at any new change, why even change this? What’s next? Renaming it from “Stars” to “Likes” just because?

The new buttons look worse too. Why do we need a badge inside the buttons like that?

And now there’s no longer a link to the fork page?

Also, these new buttons (when clicking Watch or the triangle part of Star) require clicking outside _two_ times to visually unfocus them‽
